webextensions/manifest/permissions.json

```
@@ -413,6 +413,26 @@
             }
           }
         },
+        "devtools": {
+          "__compat": {
+            "description": "<code>devtools</code>",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "54"
```

```suggestion
                "version_added": "77"
```

The permission was introduced in version 77 as a consequence of supporting optional permissions.

Before Firefox 77 (including back to 54) and in Chromium browsers, the devtools namespace is available when devtools_page is used. Would it make sense to add a note to emphasize that the permission is implicitly granted when `devtools_page` is present? Or does the mdn/content addition suffice?

---

> Would it make sense to add a note to emphasize that the permission is implicitly granted when `devtools_page` is present? Or does the mdn/content addition suffice?

I think that the details in the mdn/content addition should provide enough information without the need to repeat as a BCD note.